scripts/lint_translations.clj

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bb
+
+(ns lint-translations
+  (:require [babashka.pods :as pods]))
+
+(pods/load-pod 'clj-kondo/clj-kondo "2023.09.07")
+(require '[pod.borkdude.clj-kondo :as kondo])
+(require '[cheshire.core :as json])
+(require '[clojure.set :as set])
+
+(defn- analyze-code
```

Comment:
should these helpers be moved to a more generic or broader in scope ns?

Reply:
Re `analyze-code` - I think for now it's probably easier just to use `kondo/run!` inside `-main` directly.

Regarding `safe-name` and `->keyword` - they are quite specific to this use case (analysing all keywords in the source code) and I'd argue that it is too early to move them to a separate namespace.

`report-issues` is possibly the one function that should be extracted, but I'd argue that, again, it is too early, as we don't know for sure exactly how we want to report problems in the future (e.g. whether we want to include the namespace and the column besides the filename and the row, etc.).